### PR TITLE
[WIP] redirect previous year fair editions

### PR DESCRIPTION
**Pull request by xtina-starr**
This is WIP because I'm sticking a pin in it to work on something with a bit higher priority. Essentially this work is to redirect previous year fair editions from search traffic to current editions of the fair when there is one that is active.

https://github.com/artsy/collector-experience/issues/159 for context.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by xtina-starr** [comment]
@mzikherman mind taking a look at this when you have a moment?

File: desktop/apps/fair/test/routes.coffee [hunk 1]
```diff
@@ -261,6 +262,47 @@ describe '#fetchFairByOrganizerYear', ->
...
+    today = moment().utc()
+
+    @fairs = [
+      fabricate('fair', start_at: new Date("2-2-2014"), id: 'the-armory-show-2014', default_profile_id: 'the-armory-show-2014'),
+      fabricate('fair', start_at: new Date("2-2-2015"), id: 'the-armory-show-2015', default_profile_id: 'the-armory-show-2015'),
+      fabricate('fair', start_at: new Date("2-2-2016"), id: 'the-armory-show-2016', default_profile_id: 'the-armory-show-2016')
+      fabricate('fair', autopublish_artworks_at: today.format(), start_at: today.add(7, 'days'), end_at: today.add(14, 'days'), id: "the-armory-show-#{today.year()}", default_profile_id: "the-armory-show-#{today.year()}")
+    ]
+
+    @req = { params: { id: "the-armory-show-#{today.year()}"} }
```
Comment (mzikherman) [inline desktop/apps/fair/test/routes.coffee]: What's this for? Looks like you override it in each spec anyway. It used `today` which caught my eye as something that might start to fail next year or something :)

**Comment by mzikherman** [comment]
Looks great! 👍 just the one minor spec question

File: desktop/apps/fair/test/routes.coffee (hunk 1 above)
Comment (xtina-starr) [inline desktop/apps/fair/test/routes.coffee]: @mzikherman up above on line 272, I'm using moments to get the timestamp at the time that the test is run so I think it should always be current. Maybe there's a better approach or at least a better name. Any suggestions?

**Pull request closed without merging**